out/main.go

```diff
@@ -37,6 +49,16 @@ func buildMessage(alert Alert, m concourse.BuildMetadata) *slack.Message {
 	return &slack.Message{Attachments: []slack.Attachment{attachment}, Channel: alert.Channel}
 }
 
+func exists(path string) bool {
```

Comment:
I'm not sure this function is needed.

I think a good way to check is just by attempting to open the file (like above) and if there are any errors, to exit early and just rely on message.

For example:

Set a local variable `msg` to `Message`. Attempt to open `MessageFile` _if the value has been set_. If fails, do not reset `msg`, else, set `msg`.

Reply:
🔥